packages/@aws-cdk/aws-glue/README.md

```diff
@@ -213,8 +236,8 @@ new glue.Table(this, 'MyTable', {
   dataFormat: glue.DataFormat.JSON,
 });
 ```
-
-By default, an S3 bucket will be created to store the table's data and stored in the bucket root. You can also manually pass the `bucket` and `s3Prefix`:
+Table names within the same database must be unique. 
+Because tables created by crawlers are named based off the data target, if there are data targets with similar names in the same database, create tables with this construct and pass table names to a crawler subsequently.
```

Comment:
This is a very long phrase for English readers. Some sentences should be split, additionally, referencing the parameters that can be provided should be kept in text.